src/views/SignIn/SignIn.js

```diff
@@ -189,43 +199,64 @@ const SignIn = (props) => {
-                    >
-                      {t('commoncomponents.button.login.static.webapp.login', 'Login with Active Directory (Static Web App)')}
-                    </Button>
+                    />
+                  </Grid>
+                  <Grid item>
                     {
                       window.location.hostname === 'localhost' &&
                         <Button
-                          className={classes.loginButton}
+                          color="primary"
                           onClick={handleAuthDevSignIn}
-                          size="large"
-                          variant="contained"
-                          data-cy="log-with-dev-account-button"
+                          data-cy="sign-in-with-dev-account-button"
                         >
                           {t('commoncomponents.button.login.dev.account.login', 'Login with Dev account')}
                         </Button>
                     }
                   </Grid>
                 </Grid>
-                <Typography color="textSecondary" variant="body1">
-                  {t('commoncomponents.text.contact.get.account', 'Please contact the application administrator to activate your account.')}
-                </Typography>
+                <Grid container spacing={1} className={classes.contact} direction="row">
+                  <Grid item>
+                    <Typography color="textSecondary" variant="caption">
+                      <Box fontWeight="fontWeightLight">
+                        {t('commoncomponents.text.contact.get.account', 'Don\'t have an account?')}
+                      </Box>
+                    </Typography>
+                  </Grid>
+                  <Grid item>
+                    <Typography color="textSecondary" variant="caption">
+                      <Box fontWeight="fontWeightBold">
+                        {t('commoncomponents.text.link.cosmotech', 'Please contact CosmoTech')}
+                      </Box>
+                    </Typography>
+                  </Grid>
+                </Grid>
               </form>
             </div>
+            <div className={classes.contentFooter}>
+              <Grid spacing={2} container direction="row" justify="center" alignItems="baseline">
+               <Grid item>
+                  <FormControl className={classes.formControl}>
+                    <Select
+                      className={classes.languageSelect}
+                      defaultValue={i18n.language}
+                      onChange={(event) => changeLanguage(event.target.value)}
+                    >
+                      <MenuItem value={'en'}>English</MenuItem>
+                      <MenuItem value={'fr'}>Français</MenuItem>
+                    </Select>
+                  </FormControl>
+                </Grid>
+                <Grid item>
+                  <Typography color="textSecondary" variant="caption">
+                    © 2021 Cosmo Tech
```

Comment:
Extract the label here (maybe partners will want to ovewrite this.
 Also, use <Trans /> from react-i18next in order to format the date from JS call instead of hard-coded value